python/2FA/requirements.txt

```diff
@@ -0,0 +1,5 @@
+flask
+bandwidth-sdk
+configparser
+jwt
+json
```

Comment:
I got this error when I tried running pip to install dependencies:

```
ERROR: Could not find a version that satisfies the requirement json (from -r requirements.txt (line 5)) (from versions: none)
ERROR: No matching distribution found for json (from -r requirements.txt (line 5))
```

From what I can tell we don't need json in the requirements.txt file